packages/vue-components/src/panels/PanelBase.js

```diff
@@ -93,14 +97,19 @@ export default {
     shouldShowHeader() {
       return (!this.localExpanded) || (!this.expandHeaderless);
     },
+    shouldShowPeek() {
+      return this.peek && !this.localExpanded;
+    },
+    getCollapsedPanelHeight() {
```

Comment:
let's name it `collapsedPanelHeight` per the examples in the vue docs instead

you may also have setters for computed properties https://vuejs.org/v2/guide/computed.html#Computed-Setter